src/Administration/Resources/app/administration/src/module/sw-product/page/sw-product-detail/index.js

```diff
@@ -498,6 +498,20 @@ Component.register('sw-product-detail', {
                     }
 
                     default: {
+                        if (response.response.data.errors.length > 0
```

Comment:
What happens when `response.response.data` is undefined?

Reply:
Is this really possible? It will only go in the default stage when there is an error. If the request is successful, it will go to the success case.

Reply:
Yes, that is possible. It could be possible that the server can´t respond. Then we don´t get full information about the error :)

Reply:
I added some additional checks, should work now :)